src/hooks/useQueryErrorHandler.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import { useHistory } from 'react-router-dom'
+import { useDispatch } from 'react-redux'
+
+import { notifyErrorMsg } from 'actionCreators'
+import { ApolloError } from '@apollo/client'
+import { receiveSignoutSuccess } from 'actionCreators'
+
+// useQueryErrorHandler handles an unknown error which is caught from a
+// query operation. If the error returned is an authentication error, it
+// signs the user out and redirects them to the sign in page, otherwise it
+// displays an alert with the error message.
+export const useQueryErrorHandler = () => {
+  const [error, setError] = React.useState<unknown>()
+  const history = useHistory()
+  const dispatch = useDispatch()
+
+  React.useEffect(() => {
+    if (error === null || error === undefined) {
```

Comment:
`if(!error) `

Reply:
Changed in [`2079e2a` (#26)](https://github.com/smartcontractkit/operator-ui/pull/26/commits/2079e2a0f719b07775cdfe89764f91e91e9ea3aa)